arangod/Graph/PathManagement/PathValidator.cpp

```diff
@@ -36,6 +36,7 @@
 
 #ifdef USE_ENTERPRISE
 #include "Enterprise/Graph/Steps/SmartGraphStep.h"
+#include "Enterprise/Graph/PathValidatorEE.cpp"
```

Comment:
```suggestion
```

Never include .cpp files.
Maybe you need to add the above cpp file to CMake. in Enterprise Repository, I have not checked that.

Reply:
Added a note, as discussed. Due to TEMPLATING we cannot solve this differently.